cmd/gaia/app/genesis.go

```diff
@@ -70,9 +69,16 @@ func GaiaAppInit() server.AppInit {
 		"home directory for the client, used for key generation")
 	fsAppGenTx.Bool(flagOWK, false, "overwrite the accounts created")
 
+	flagNames := server.GenTxFlagNames{
+		FlagName:       flagName,
+		FlagClientHome: flagClientHome,
+		FlagOWK:        flagOWK,
+	}
```

Comment:
I thought about that a lot, here's the reason behind it:
 I want to make sure we only define flag names in one place. This used to be the server package, `init.go`. The problem is that now, because of the testnet command, I need to hand over the parameters through AppInit which is in turn used in the app package (genesis.go). In the genesis.go file I don't have access to the flag names if they are set as global variables. If they are passed over like this, I can query viper for their value.